src/Mockable/Concurrent.hs

```diff
@@ -50,29 +47,16 @@ myThreadId = liftMockable MyThreadId
 killThread :: ( Mockable Fork m ) => ThreadId m -> m ()
 killThread tid = liftMockable $ KillThread tid
 
+type RelativeToNow = Microsecond -> Microsecond
```

Comment:
I guess we don't need this and `for` anymore, can just make `delay :: TimeUnit t => t -> m ()`